.github/workflows/superset-python-misc.yml

```diff
@@ -86,7 +86,12 @@ jobs:
           echo "HOMEBREW_REPOSITORY=$HOMEBREW_REPOSITORY" >>"${GITHUB_ENV}"
           brew install norwoodj/tap/helm-docs
       - name: pre-commit
-        run: pre-commit run --all-files
```

Comment:
@jsoref would the `pre-commit run` `--show-diff-on-failure` flag be suffice?

Reply:
Maybe. I don't use this tool. As long as I can apply the diff and get it not to fail going forward.

Reply:
Seems like a good idea. I don't know much about pre commit tbh